apps/designer/app/designer/features/style-panel/controls/color/color-control.tsx

```diff
@@ -1,29 +1,59 @@
 import { Flex } from "@webstudio-is/design-system";
+import { toValue } from "@webstudio-is/css-engine";
 import { getFinalValue } from "../../shared/get-final-value";
 import type { ControlProps } from "../../style-sections";
 import { ColorPicker } from "../../shared/color-picker";
+import { colord } from "colord";
 
 export const ColorControl = ({
   currentStyle,
   inheritedStyle,
   setProperty,
   styleConfig,
 }: ControlProps) => {
-  const value = getFinalValue({
+  let value = getFinalValue({
     currentStyle,
     inheritedStyle,
     property: styleConfig.property,
   });
+
   if (value === undefined) {
     return null;
   }
+
   const setValue = setProperty(styleConfig.property);
 
+  if (value.type !== "rgb") {
+    // Support previously set colors
+    const colordValue = colord(toValue(value));
```

Comment:
I guess this is fine, but we could also write a migration script and then remove it?

Reply:
also for now I would add an example of a value we know is in the database

Reply:
a reader doesn't know it was previously saved as a keyword

Reply:
Will be hard to convert keyword to rgb, we will need to know all "color" properties

Reply:
actually no, there are more, but the list is not super long, our search shows you them all:
<img width="239" alt="Screenshot 2022-12-09 at 15 32 47" src="https://user-images.githubusercontent.com/52824/206737227-9a7ac592-22ac-429b-a16a-490577fb71ef.png">

Reply:
since we don't use shorthands, all color properties are easy to identify

Reply:
but its def. separate pr